src/composables/useFlawCvssScores.ts

```diff
@@ -92,9 +92,14 @@ export function useFlawCvssScores(flaw: Ref<ZodFlawType>) {
 
   async function saveCvssScores() {
     if (flawRhCvss3.value?.created_dt) {
+      // Handle existing CVSS score
+      if (flawRhCvss3.value?.vector === null && flawRhCvss3.value?.uuid !== null) {
```

Comment:
```suggestion
      if (flawRhCvss3.value?.vector === null && flawRhCvss3.value?.uuid != null) {
```

Reply:
`!=` will allow undefined, which likely would be the value if the cvss data were not saved